test/integration/test_b2_command_line.py

```diff
@@ -109,10 +109,10 @@ def read_from(self, f):
             self.string = str(e)
 
 
-def remove_insecure_platform_warnings(text):
+def remove_warnings(text):
```

Comment:
do we really need it? it was for python 2.7

Reply:
We have to remove `DeprecationWarning` from `arrow` and `python3.5`. I don't know about `SNIMissingWarning` and `InsecurePlatformWarning`. When we use pytest, it can be implemented better and we can use pytest built-in feature for hiding particular warning per test, but not in this PR.

Reply:
yeah those two were for python 2.7, lets remove them

Reply:
Will remove